### PR TITLE
remove drop cap style from paragraphs

### PR DESCRIPTION
**Pull request by jivey**
Before:
after

After:
before

### CODE REVIEW

File: tutor/resources/styles/book-content/base.scss [hunk 1]
```diff
@@ -63,7 +63,6 @@
   section:not(.references) h1 + p:first-of-type,
   [data-type="document-title"] ~ p:first-of-type {
     margin-top: 2rem;
-    @include tutor-reading-first-letter();
   }
```
Comment (nathanstitt) [inline tutor/resources/styles/book-content/base.scss]: Awesome!  can you also do a search for `tutor-reading-first-letter` and remove the mixin if it's not in use anywhere else? I suspect this was the only spot it was referenced.

Comment (jivey) [reply tutor/resources/styles/book-content/base.scss]: Good call, this was unreferenced and unlikely to be used in the future. Removed in a follow up commit 👍

**Comment by nathanstitt** [comment]
not a big deal on this one since what's happening is entirely self-explanatory, but typically for anything that has visual changes we like to include a before and after screenshot on the PR.

**Comment by jivey** [comment]
Agree, updated in the PR description!

**Pull request merged**